deliverable_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+read -p "Which CI/CD service do you use (Can be edited later) [(g)ithub/(b)itrise]: " ciService
```

Comment:
How about adding `later` option?
```suggestion
read -p "Which CI/CD service do you use (Can be edited later) [(g)ithub/(b)itrise/(l)ater]: " ciService
```

Reply:
Sounds good, better UX that way.

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/320/commits/9802c59357c1203fdcc9b21e5af0f57f0109237a